### ISSUE TITLE
Win11 Exception  Unmapped relationship: 7

### ISSUE THREAD

**Issue by linG5821** [didi/KnowStreaming#527]
The project used the oshi-core dependency, but the oshi-core version reported an error in Windows 11. see oshi/oshi#1554 oshi/oshi#1566  This bug is fixed with 5.6.1.
I have tried to update the version and the boot is no longer abnormal, but I am not sure if there are other problems.

branch: master
file: km-task/pom.xml

```
<dependencies>
        <dependency>
            <groupId>io.github.zqrferrari</groupId>
            <artifactId>logi-job-spring-boot-starter</artifactId>
            <exclusions>
                <exclusion>
                    <artifactId>oshi-core</artifactId>
                    <groupId>com.github.oshi</groupId>
                </exclusion>
            </exclusions>
        </dependency>

        <dependency>
            <groupId>com.github.oshi</groupId>
            <artifactId>oshi-core</artifactId>
            <version>5.6.1</version>
        </dependency>
</dependencies>
```

**Comment by ZQKC** [comment]
> The project used the oshi-core dependency, but the oshi-core version reported an error in Windows 11. see [oshi/oshi#1554](https://github.com/oshi/oshi/issues/1554) [oshi/oshi#1566](https://github.com/oshi/oshi/pull/1566) This bug is fixed with 5.6.1. I have tried to update the version and the boot is no longer abnormal, but I am not sure if there are other problems.
> 
> branch: master file: km-task/pom.xml
> 
> ```
> <dependencies>
>         <dependency>
>             <groupId>io.github.zqrferrari</groupId>
>             <artifactId>logi-job-spring-boot-starter</artifactId>
>             <exclusions>
>                 <exclusion>
>                     <artifactId>oshi-core</artifactId>
>                     <groupId>com.github.oshi</groupId>
>                 </exclusion>
>             </exclusions>
>         </dependency>
> 
>         <dependency>
>             <groupId>com.github.oshi</groupId>
>             <artifactId>oshi-core</artifactId>
>             <version>5.6.1</version>
>         </dependency>
> </dependencies>
> ```

Thanks for your feedback! 
We will try to upgrade oshi-core which in logi-job-spring-boot-starter, and you can solve the problem in your ways now.

**Comment by linG5821** [comment]
Thanks I will close this issue
